### PR TITLE
Add HomePage Link to README and shorten it.

### PR DESCRIPTION
**Pull request by DanVanAtta**
## Overview

There is a bit too much information on the README page that turns it
into mostly noise. The updates here are to clarify the headers and
remove content.

The license block was large and simply extracted, the game description
list is removed as we go over that on the homepage. The page beginning
with a link to download did not 100% make sense and made scanning a bit
difficult. Instead we now start with a link to the homepage.

Last, the availaibility report title was updated to make it more obvious
what it is.

### Before
screenshot from 2019-01-05 12-23-09

### After
screenshot from 2019-01-05 12-22-54

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,48 +4,25 @@
...
 
 ## Documentation Links
-- Developer, getting started: https://github.com/triplea-game/triplea/tree/master/docs/dev
-- Moderator, how-to: https://forums.triplea-game.org/ (secret admin section)
-- Setting up Linode servers: https://github.com/triplea-game/lobby#adding-linode-servers
-- Lobby+Bot Ops (install, check status, restart): https://github.com/triplea-game/lobby
-- Forums Ops: https://github.com/triplea-game/triplea/tree/master/docs/admin#nodebb-forums
-- Dice Server (TODO): https://github.com/triplea-game/triplea/tree/master/docs/admin#the-dice-server
-
-## License
```
Comment (ssoloff) [inline README.md]: Removing this heading will require an update to the license notice URL:

https://github.com/triplea-game/triplea/blob/f43cfdd8d71e9e73d9e76f90c334925b8deac1f0/game-core/src/main/java/games/strategy/triplea/UrlConstants.java#L45

File: docs/license.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[Full license](https://github.com/triplea-game/triplea/blob/master/LICENSE)
```
Comment (ssoloff) [inline docs/license.md]: Probably dangerous to call the target document the "full license," as the notice below encapsulates our complete license agreement due to the exception we had to grant for non-GPL-compliant libraries.  If you want to provide this link, I would recommend simply hyperlinking the phrase `GNU General Public License` on line 12 below.  However, the _LICENSE_ file is identical to what's available on `gnu.org` (a link to which also appears below), so it's probably not necessary to link it twice.

Comment (DanVanAtta) [reply docs/license.md]: Backing up, do you know if we need a LICENSE file at all? Could we just link to the `gnu.org` copy?

File: README.md (hunk 1 above)
Comment (DanVanAtta) [reply README.md]: Which also means we'll break existing clients  :face_with_head_bandage: 

Good catch. 

The README.md is/was too heavy. @ssoloff beyond what was done here, can you suggest alternatives or more things that can be done to cleanly tuck 'license' away to another page and reduce redundant material?

File: docs/license.md (hunk 2 above)
Comment (ssoloff) [reply docs/license.md]: Primarily, _LICENSE_ permits GitHub to deduce the repo's license for display:

license

Secondarily, I guess, it's just convention for OSS projects to have a canonical location in which to find the project's license.  The [GitHub docs](https://help.github.com/articles/licensing-a-repository/#determining-the-location-of-your-license) simply state (emphasis mine):

> Most people place their license text in a file named LICENSE.txt (or LICENSE.md) in the root of the repository; here's an example from Hubot.
>
> Some projects include information about their license in their README. For example, a project's README may include a note saying "This project is licensed under the terms of the MIT license."
>
> **As a best practice, we encourage you to include the license file with your project.**

Comment (DanVanAtta) [reply docs/license.md]: In the meantime, I'm happy to update the link text to not say "full license", done in: b7164b4fc

Comment (ssoloff) [reply docs/license.md]: > I'm wondering if we could simply add our additional grants to the top of the existing LICENSE file

I believe I tried that when we upgraded to GPLv3.  IIRC, GitHub could no longer deduce our license, as evidenced by the red box I illustrated above being empty.

**Pull request merged**